cmd/pipeline/main.go

```diff
@@ -72,13 +72,23 @@ func initLog() *logrus.Entry {
 }
 
 func main() {
-	if len(os.Args) > 1 && os.Args[1] == "--version" {
-		if CommitHash == "" {
-			fmt.Println("version: ", Version, " built on ", BuildDate)
-		} else {
-			fmt.Printf("version: %s-%s built on %s\n", Version, CommitHash, BuildDate)
+	if len(os.Args) > 1 {
+		if os.Args[1] == "--version" {
+			if CommitHash == "" {
+				fmt.Println("version: ", Version, " built on ", BuildDate)
+			} else {
+				fmt.Printf("version: %s-%s built on %s\n", Version, CommitHash, BuildDate)
+			}
+			os.Exit(0)
+		}
+
+		if os.Args[1] == "sync" {
```

Comment:
How is it going to run exactly? In a separate pod as a cron job?

For now it would be better to keep everything inside the pipeline process (if possible)